mod-source-record-manager-server/src/main/resources/templates/db_scripts/create_rule_migration_change_log_table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS rule_migration_change_log (
```

Comment:
You should also populate the table with migrations that are expected to already be executed before this PR.